man/ctags-lang-terraform.7.rst.in

```diff
@@ -34,12 +34,6 @@ to extract variables assigned in variables definitions (`*.tfvars`).
 The TerraformVariables parser extracts variables in `*.tfvars` files
 with ``variable`` kind with ``assigned`` role of ``Terraform`` language.
 
-KNOWN BUGS
-----------
-* The parser doesn't ignore strings inside a block comment (``/* ... */``).
-
-* The parser doesn't extract variables defined with ``locals`` keyword.
-
```

Comment:
Thank you for updating the manual page.

Could you add:

```
VERSIONS
--------

Change since "0.0"
~~~~~~~~~~~~~~~~~~

* New kind ``local``
```